### PR TITLE
ansible playbook and variables for Qld nectar GPU pulsar

### PR DESCRIPTION
**Pull request by jlqfab**

### CODE REVIEW

File: host_vars/pulsar-qld-gpu1.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ssl_country: "AU"
+ssl_state: "Qld"
+ssl_location: "Brisbane"
+ssl_organisation: "Galaxy Australia"
+ssl_operational_unit: "Pulsar Qld GPU1"
+ssl_email: "help@genome.edu.au"
+
+#host specific pulsar settings
+
+rabbitmq_password_galaxy_au: "{{ vault_rabbitmq_password_galaxy_hm1_prod }}"
```
Comment (cat-bro) [inline host_vars/pulsar-qld-gpu1.yml:15]:
```suggestion
rabbitmq_password_galaxy_au: "{{ vault_rabbitmq_password_galaxy_qldgpu1_prod }}"
```

**Comment by jlqfab** [comment]
Not ready to merge. I think there are additional variables to define, and I've already noticed a line that I forgot to update from phm1's version.

File: host_vars/pulsar-qld-gpu1.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+pulsar_queue_url: "galaxy-queue.usegalaxy.org.au"
+pulsar_rabbit_username: "galaxy_hm1"
```
Comment (cat-bro) [inline host_vars/pulsar-qld-gpu1.yml:17]:
```suggestion
pulsar_rabbit_username: "galaxy_qldgpu1"
```

File: host_vars/pulsar-qld-gpu1.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+pulsar_rabbit_vhost: "/pulsar/galaxy_hm1"
```
Comment (cat-bro) [inline host_vars/pulsar-qld-gpu1.yml:18]:
```suggestion
pulsar_rabbit_vhost: "/pulsar/galaxy_qldgpu1"
```

**Comment by cat-bro** [comment]
this probably an `attached_volumes` with nvme0n1 going to /mnt

**Review by cat-bro — APPROVED** [review]
looks good to me

File: pulsar-qld-gpu1_playbook.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
+- hosts: pulsar_qld_gpu1
```
Comment (cat-bro) [inline pulsar-qld-gpu1_playbook.yml:1]:
```suggestion
- hosts: pulsar-qld-gpu1
```

Comment (jlqfab) [reply pulsar-qld-gpu1_playbook.yml]: Thanks for picking that up.

**Comment by cat-bro** [comment]
looks good to me. I can add some galaxy-side things

**Review by cat-bro — DISMISSED** [review]

**Comment by cat-bro** [comment]
could you please add passwords for vault_rabbitmq_password_galaxy_qldgpu2_prod thru vault_rabbitmq_password_galaxy_qldgpu6_prod in this PR? easier than having several PRs that modify vault

**Review by cat-bro — APPROVED** [review]

**Pull request merged**